src/compiler/checker.ts

```diff
@@ -3676,6 +3690,12 @@ module ts {
             return getTypeOfGlobalSymbol(getGlobalTypeSymbol(name), arity);
         }
 
+        function getNamespacedType(namespace: string, name: string): Type {

```

Comment:
What about getExportedType? I don't like the use of "namespaced" as an adjective.